docs/installFeature.md

```diff
@@ -3,6 +3,9 @@ The `installFeature` task installs features packaged as a Subsystem Archive (ESA
   
 This goal is not supported if the Liberty runtime is installed from the Open Liberty runtime package. You will get a missing file error since the `bin/installUtiltiy` command is removed from the Open Liberty runtime package. The Open Liberty runtime is always bundled with all applicable features and there isn't any need to install any additional feature.  
 
+### dependsOn
+`installFeature` depends on `libertyCreate` if there are any features to install into a server, otherwise `installLiberty`.
```

Comment:
Doesn't this one depend on `libertyCreate if there are no specific features requested so that the server.xml is evaluated? ... otherwise `installLIberty` ?